ee/maintained-apps/ingesters/homebrew/external_refs/microsoft_version_from_release_notes.go

```diff
@@ -0,0 +1,47 @@
+package externalrefs
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+
+	macoffice "github.com/fleetdm/fleet/v4/server/vulnerabilities/macoffice"
+)
+
+var (
+	releaseNotesCache macoffice.ReleaseNotes
+	cacheOnce         sync.Once
+)
+
+// MicrosoftVersionFromReleaseNotes returns the short version from the release notes given a Homebrew version.
+// Homebrew version "16.95.25032931"
+// Release notes version "Version 16.95.3 (Build 25032931)"
+// For this example it would return "16.95.3"
+func MicrosoftVersionFromReleaseNotes(args ...interface{}) (string, error) {
```

Comment:
This feels like we should spec this as a mutation function that operates on...outputs, maybe (similar to CVE feed tweaks)...rather than as something with untyped args. We can also index overrides by slug rather than needing to specify function names inside the code (maybe gated behind a `has_custom_logic` flag on the input file so it's clear that we're doing something special on a given input).